### PR TITLE
Core modul implemented

### PR DESCRIPTION
**Pull request by Trizax350**
Classes are created and core modul is implemented.

### CODE REVIEW

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Courses.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+public class Courses {
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Courses.java]: Please consider renaming the class to the singular noun

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Complains.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+public class Complains {
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Complains.java]: Please consider renaming the class to the singular noun

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+public class Requests {
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java]: Please consider renaming the class to the singular noun

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/ConnectionUsersAndCourses.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+//This class is for represent database many-many connection between users and courses
+public class ConnectionUsersAndCourses {
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/ConnectionUsersAndCourses.java]: This is not a database, use collections and services

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    private int id;
+    private String priority;
+    private String type;
+    private String description;
+    private String course_name;
+    private String statusz;
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java]: status

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java [hunk 6]
```diff
@@ -0,0 +1,79 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+public class Requests {
+    private int id;
+    private String priority;
+    private String type;
+    private String description;
+    private String course_name;
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Requests.java]: Probably a reference to the Course would be a better solution

File: core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Users.java [hunk 7]
```diff
@@ -0,0 +1,42 @@
+package iit.uni.miskolc.hu.softwaretesting.model;
+
+public class Users {
+    private int id;
+    private String name;
+    private String email;
+    private String username;
+    private String password;
+    private String type;
```
Comment (judit0310) [inline core/src/main/java/iit/uni/miskolc/hu/softwaretesting/model/Users.java]: What is the type for?

**Comment by Trizax350** [comment]
I think I fixed all of them. Please check it.

**Pull request closed without merging**